sidechain/storage/src/storage_tests_get_blocks_following.rs

```diff
@@ -0,0 +1,105 @@
+/*
```

Comment:
This should probably be renamed now too? Since the method is now called `get_blocks_after()`.. this includes the source file name and the test names

Reply:
Good catch, thanks.